buildSrc/build.gradle.kts

```diff
@@ -1,5 +1,11 @@
 plugins {
-    kotlin("jvm") version "1.5.21"
+    kotlin("jvm") version "1.6.0"
+}
+
+java {
+    toolchain {
+        languageVersion.set(JavaLanguageVersion.of(11))
+    }
 }
```

Comment:
Please use Java 8 in all 3 occurrences. We want the scripts to be Java 8 comaptible, just like the reast of BT and GE.

Reply:
Sounds good, I've updated the Github action workflows files accordingly